<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="test.java.NewTest" duration-ms="22729" started-at="2015-05-07T13:57:36Z" finished-at="2015-05-07T13:57:59Z">
    <groups>
    </groups>
    <test name="Command line test" duration-ms="22729" started-at="2015-05-07T13:57:36Z" finished-at="2015-05-07T13:57:59Z">
      <class name="test.java.NewTest">
        <test-method status="PASS" signature="BeforeTest()[pri:0, instance:test.java.NewTest@32e6e9c3]" name="BeforeTest" is-config="true" duration-ms="259" started-at="2015-05-07T16:57:36Z" finished-at="2015-05-07T16:57:37Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- BeforeTest -->
        <test-method status="PASS" signature="setUp(java.lang.reflect.Method)[pri:0, instance:test.java.NewTest@32e6e9c3]" name="setUp" is-config="true" duration-ms="8466" started-at="2015-05-07T16:57:37Z" finished-at="2015-05-07T16:57:45Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void test.java.NewTest.testAddDevice() throws java.lang.Exception]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="testAddDevice()[pri:0, instance:test.java.NewTest@32e6e9c3]" name="testAddDevice" duration-ms="12945" started-at="2015-05-07T16:57:45Z" finished-at="2015-05-07T16:57:58Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.fail(Assert.java:101)
	at test.java.NewTest.testDeleteDevice(NewTest.java:229)
	at test.java.NewTest.testAddDevice(NewTest.java:193)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:696)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:882)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1189)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:110)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
	at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testAddDevice -->
        <test-method status="PASS" signature="takeScreenShotOnFailure(org.testng.ITestResult, java.lang.reflect.Method)[pri:0, instance:test.java.NewTest@32e6e9c3]" name="takeScreenShotOnFailure" is-config="true" duration-ms="615" started-at="2015-05-07T16:57:58Z" finished-at="2015-05-07T16:57:59Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=testAddDevice status=FAILURE method=NewTest.testAddDevice()[pri:0, instance:test.java.NewTest@32e6e9c3] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void test.java.NewTest.testAddDevice() throws java.lang.Exception]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- takeScreenShotOnFailure -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:test.java.NewTest@32e6e9c3]" name="tearDown" is-config="true" duration-ms="430" started-at="2015-05-07T16:57:59Z" finished-at="2015-05-07T16:57:59Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- test.java.NewTest -->
    </test> <!-- Command line test -->
  </suite> <!-- test.java.NewTest -->
</testng-results>
